src/test/java/com/google/sps/tool/GoogleTranslateTest.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.tool;
+
+import org.junit.Test;
+import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.google.sps.tool.GoogleTranslate;
+
+public class GoogleTranslateTest {
+
+  @Test
+  public void translateHelloToSpanish() {
+    assertEquals("Hola", GoogleTranslate.translateText("hello", "es"));
```

Comment:
i wonder for testing this is not whether google provides an accurate translation but that function correctly calls the translation API and handles its response. In other words, the tests should be independent from what the outside service returns.